Systemen.tex

```diff
@@ -53,12 +53,16 @@ \section*{Build information}
 The most recent version of this document can be found on: \href{https://c7878.gitlab.io/systemen/Systemen.pdf}{https://c7878.gitlab.io/systemen/Systemen.pdf}
 \section*{Introduction}
 
-These are the derivations for maths for systems. \href{https://github.com/debber1/Systemen}{The source code for this document can be found on github.} 
+These are the derivations for mathematical systems. \href{https://github.com/debber1/Systemen}{The source code for this document can be found on github.} 
```

Comment:
This is a good correction, but we still have to change the title of the document to reflect the proper name of this course